integration/conftest.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import fakeredis
+import redis as r
+import pytest
+
+from storey import RedisDriver
+
+REDIS_ENDPOINT = os.environ.get('REDIS_URL')
```

Comment:
If it's a URL why not call it `REDIS_URL`? Or the other way around if it's an endpoint.

Reply:
I'm not really sure what is the difference between URL, API or ENDPOINT :-)
mlrun use API - do you want to settle on that?

Reply:
fixed